packages/braid-design-system/src/lib/components/private/Field/Field.tsx

```diff
@@ -25,19 +25,16 @@ export type FieldLabelVariant =
       'aria-labelledby': string;
       secondaryLabel?: never;
       tertiaryLabel?: never;
-      description?: never;
     }
   | {
       'aria-label': string;
       secondaryLabel?: never;
       tertiaryLabel?: never;
-      description?: never;
     }
   | {
       label: FieldLabelProps['label'];
       secondaryLabel?: FieldLabelProps['secondaryLabel'];
       tertiaryLabel?: FieldLabelProps['tertiaryLabel'];
-      description?: FieldLabelProps['description'];
```

Comment:
Out of scope for this PR, but would there be any value in displaying these prop type variants more explicitly on the component docs pages? Currently `label`, `aria-label` and `aria-labelledby` are all displayed as `Required`, when in reality only one of them is required. I'm guessing this an artifact of how the type contract serializer compresses these union types.

![image](https://user-images.githubusercontent.com/5663042/222995183-a1c7d3ac-8ba8-425e-bba6-446b9ca81cc7.png)

Reply:
Yeah we can pick this up, there are current limitations to the type extraction script.